### PR TITLE
New formula Qore

### PR DESCRIPTION
**Pull request by Vykook**

### CODE REVIEW

File: Library/Formula/qore.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Qore < Formula
+  desc "multi-threaded embedable scripting language"
+  homepage "http://qore.org"
+  url "https://downloads.sourceforge.net/project/qore/qore/0.8.11/qore-0.8.11.tar.bz2"
+  sha256 "ce55fe2fa2bb85fd77cefe45d190d431fa5c6dcbd00cc4be499082130b755ae3"
+
+  depends_on "flex"
```
Comment (DomT4) [inline Library/Formula/qore.rb]: Is this incapable of using the system `flex` and `bison` versions?

**Comment by davidnich** [comment]
as the author of the Qore programming language I can confirm that qore needs flex 2.5.31 or above, a requirement which is satisfied in Yosemite, but not in previous releases of OSX.

I believe that no special version of bison is needed on OSX though.

**Comment by DomT4** [comment]
> as the author of the Qore programming language I can confirm

Thanks for commenting!

@mikemcquaid What do we want to do here? `depends_on "flex" if MacOS.version < :yosemite` or just leave it unconditional?

@Vykook Let's cut the `bison` dependency per @davidnich's feedback.

**Comment by mikemcquaid** [comment]
> @mikemcquaid What do we want to do here? depends_on "flex" if MacOS.version < :yosemite or just leave it unconditional?

The `if` version is :+1:

**Comment by mistydemeo** [comment]
Is the flex dep needed at runtime, or is it only needed at buildtime? You can make it `depends_on "flex" => :build` if the latter.

**Comment by davidnich** [comment]
build-time only

File: Library/Formula/qore.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Qore < Formula
+  desc "multi-threaded embedable scripting language"
```
Comment (dunn) [inline Library/Formula/qore.rb]: Typo: "embedable" should be "embeddable."

Comment (DomT4) [inline Library/Formula/qore.rb]: Start with a capital letter as well please. `Multi-threaded`

**Comment by DomT4** [comment]
If you could update to address the various comments in this thread that'd be great, Thanks.

**Comment by dunn** [comment]
I don't think we're going to merge this, for notability reasons; see #43508.

Thank you for the submission, though!

**Pull request closed without merging**